### PR TITLE
Dailymotion Bid Adapter: accept ortb2 field

### PR DESCRIPTION
**Pull request by sebmil-daily**
## Type of change

- [x] Other : docs PR updated (prebid.github.io) -> https://github.com/prebid/prebid.github.io/pull/5091

## Description of change

This addresses the comments that we received after https://github.com/prebid/Prebid.js/pull/10970 was merged.

The main changes are:
* use standard ORTB video parameters as fallback
* support several new ORTB video fields
* better request validation to exclude non video/instream requests
* preliminary support for in-app requests

### CODE REVIEW

File: modules/dailymotionBidAdapter.js [hunk 1]
```diff
@@ -38,17 +45,22 @@ function getVideoMetadata(bidRequest, bidderRequest) {
   const videoMetadata = {
     description: videoParams.description || '',
     duration: videoParams.duration || 0,
-    iabcat1: Object.keys(iabcat1),
+    iabcat1: Array.isArray(videoParams.iabcat1)
```
Comment (karimMourra) [inline modules/dailymotionBidAdapter.js:48]: ortb content has `cat` see section 5.1 
Per spec:
cat - string array - Array of IAB content categories that describe the content
producer. Refer to List 5.1.

File: modules/dailymotionBidAdapter.md [hunk 2]
```diff
@@ -102,15 +116,18 @@ const adUnits = [
...
         id: '54321',
         lang: 'FR',
-        playerSize: [ [1280, 720] ],
+        livestream: 0,
         private: false,
-        startDelay: 0,
+        startdelay: 0,
         tags: 'tag_1,tag_2,tag_3',
         title: 'test video',
         topics: 'topic_1, topic_2',
```
Comment (karimMourra) [inline modules/dailymotionBidAdapter.md:128]: Please exclude fields that are not defined in `imp[].video` from `mediaTypes.video`. 
The direction we are moving is to align `imp[].video` with `mediaTypes.video` to simplify onboarding for publishers. Most publishers use multiple SSPs; by standardizing the configuration across the board, we can reduce the complexity of getting setup with video in Prebid. 

Everything outside of the spec can be set in `params.video`

File: modules/dailymotionBidAdapter.md [hunk 3]
```diff
@@ -122,15 +139,29 @@ If a field exists in both places, it will be overridden by bids.params.video.
```
Comment (karimMourra) [inline modules/dailymotionBidAdapter.md:139]: Please edit the following statement:

> Each of the following video metadata fields can be added in mediaTypes.video or bids.params.video.

Only fields supported in `imp[].video` should be defined in `mediaTypes.video`

**Review by karimMourra — CHANGES_REQUESTED** [review]

File: modules/dailymotionBidAdapter.js (hunk 1 above)
Comment (kvnsw) [reply modules/dailymotionBidAdapter.js:47]: Added fallback to `cat`

File: modules/dailymotionBidAdapter.md (hunk 2 above)
Comment (kvnsw) [reply modules/dailymotionBidAdapter.md:128]: Dropped all support of non standard fields in `mediaTypes.video`. Updated documentation accordingly

**Comment by sebmil-daily** [comment]
Thanks @karimMourra for all the suggestions. I think @kvnsw adressed all of them, could you confirm please ?

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
LGTM

@karimMourra if you have additional feedback let's handle in subsequent pr's

**Comment by karimMourra** [comment]
Apologies, I lost track of this PR. The changes look great thank you for addressing!
Swing by the Video Taskforce sync sometime! We meet every 2 weeks on Thursdays at 12PM EST. We are meeting this week.

**Pull request merged**